wire-gradle-plugin/api/wire-gradle-plugin.api

```diff
@@ -146,7 +146,7 @@ public final class com/squareup/wire/gradle/WireExtension$ProtoRootSet {
 	public final fun srcJar (Lorg/gradle/api/provider/Provider;)V
 	public final fun srcJar (Lorg/gradle/api/provider/ProviderConvertible;)V
 	public final fun srcProject (Ljava/lang/String;)V
-	public final fun srcProject (Lorg/gradle/api/internal/catalog/DelegatingProjectDependency;)V
```

Comment:
Hmmm. `ProjectDependency` is a superclass of `DelegatingProjectDependency` and is mostly used in Gradle DSL. I cannot really imagine someone break with this change. In what scenario would that happen?

Reply:
Gradle plugin A (say a convention plugin) is configuring wire, and is compiled against v5.0 of the wire gradle plugin. It is then applied in a build where the consumer is pulling in wire v5.2 (or whatever this change goes into). At runtime it will get a MethodNotFound error because it was compiled against a function with the old signature.

Reply:
Got it. I put back the old method as well